app/Exports/OptionExport.php

```diff
@@ -0,0 +1,189 @@
+
+    /**
+     * Sheet name to export.
+     *
+     * @var string
+     */
+    protected string $sheetName;
+
+    /**
+     * Configuration for color code and cell merge for different xls files.
+     *
+     * @var array|array[]
+     */
+    protected array $instruction_properties = [
+      'period_instructions' => [
+          'color_code' =>[
+              'C6' => 'd5a6bd',
+              'C7' => '93c47d',
+              'C8' => 'f6b26b',
+              'C9' => '6fa8dc',
+          ],
+          'merge_cells' => [
+              'B1:C1',
+              'B2:C2',
+              'B3:C3',
+              'B4:C4',
+              'B5:C5',
+              'B10:C10',
+              'B11:C11',
+              'B12:C12',
+              'B13:C13',
+              'B14:C14',
+          ],
+      ],
+      'activity_instructions' => [
+          'color_code' => [
+              'C4' => 'd5a6bd',
+              'C5' => '93c47d',
+              'C6' => 'f6b26b',
+              'C7' => '6fa8dc',
+          ],
+          'merge_cells' => [
+              'B1:C1',
+              'B2:C2',
+              'B3:C3',
+              'B8:C8',
+              'B9:C9',
+              'B10:C10',
+              'B11:C11',
+              'B12:C12',
+              'B13:C13',
+          ],
+      ],
+      'result_instructions' => [
+          'color_code' =>[
+              'C5' => 'd5a6bd',
+              'C6' => '93c47d',
+              'C7' => 'f6b26b',
+              'C8' => '6fa8dc',
+          ],
+          'merge_cells' => [
+              'B1:C1',
+              'B2:C2',
+              'B3:C3',
+              'B4:C4',
+              'B9:B9',
+              'B10:C10',
+              'B11:C11',
+              'B12:C12',
+              'B13:C13',
+          ],
+      ],
+      'indicator_instructions' => [
+          'color_code' =>[
+              'C6' => 'd5a6bd',
+              'C7' => '93c47d',
+              'C8' => 'f6b26b',
+              'C9' => '6fa8dc',
+          ],
+          'merge_cells' => [
+              'B1:C1',
+              'B2:C2',
+              'B3:C3',
+              'B4:C4',
+              'B5:C5',
+              'B10:C10',
+              'B11:C11',
+              'B12:C12',
+              'B13:C13',
+          ],
+      ],
+    ];
+
+    /**
+     * @param $fileName
+     * @param $sheetName
+     */
+    public function __construct($fileName, $sheetName)
+    {
+        $this->fileName = $fileName;
+        $this->sheetName = $sheetName;
+    }
+
+    /**
+     * To define the name of a sheet.
+     *
+     * @return string
+     */
+    public function title(): string
+    {
+        return $this->sheetName;
+    }
+
+    /**
+     * To export data using blade file.
+     *
+     * @throws JsonException
+     *
+     * @return View
+     */
+    public function view(): View
+    {
+        $data = readJsonFile('Exports/XlsExportTemplate/' . $this->fileName . '.json');
+
+        return view('Export.optionExport', ['data' => $data]);
```

Comment:
use readJsonFile('Exports/XlsExportTemplate/' . $this->fileName . '.json') directly instead of declaring and using $data